mmcv/runner/fp16_utils.py

```diff
@@ -346,10 +360,10 @@ class LossScaler:
     """
 
     def __init__(self,
-                 init_scale=2**32,
-                 mode='dynamic',
-                 scale_factor=2.,
-                 scale_window=1000):
+                 init_scale: float = 2**32,
```

Comment:
The type of default value is `int`

Reply:
![image](https://user-images.githubusercontent.com/92794867/170829143-182fa905-89bf-4c3e-b6cc-cbd3f58a3537.png)
Just the same as doc

Reply:
It could be better to set default value for `init_scale`: `init_scale: float = 2.**32`